tests/unit/vault/test_profit_unlocking.py

```diff
@@ -2449,3 +2421,73 @@ def test_loss_fees_refunds_with_buffer(
         total_assets=0,
```

Comment:
This line at 2217 https://github.com/jmonteer/yearn-vaults-v3/blob/3ee50c8675183895ab5477a2c08991d598167bfc/tests/unit/vault/test_profit_unlocking.py#L2217 should use 31_556_952 instead of 365 * 24 * 3600 since that is the hardcoded seconds_per_year that the mock flexible accountant is using to calculate the fee